Issue on page /03-how-to-package-a-python.html -- No module named 'autoapi'

When trying to render the documentation, I encountered the following error:
```python
cd docs
poetry run make html

```
```console
Running Sphinx v3.5.4
Extension error:
Could not import extension autoapi.extension (exception: No module named 'autoapi')
make: *** [Makefile:19: html] Error 2
```

I solved the issue by adding `sphinx-autoapi` to the pyproject.toml file.
```python
poetry add --dev sphinx nbsphinx ipykernel sphinx-autoapi
```

--- a/py-pkgs/03-how-to-package-a-python.ipynb
+++ b/py-pkgs/03-how-to-package-a-python.ipynb
@@ -558,7 +558,7 @@
     "First, we need to install `sphinx`, `nbsphinx`, and `ipykernel` as development dependencies using `poetry`. \n",
     "\n",
     "```{prompt} bash\n",
-    "poetry add --dev sphinx nbsphinx ipykernel\n",
+    "poetry add --dev sphinx nbsphinx ipykernel sphinx-autoapi\n",
     "```\n",
     "\n",
     "```{note}\n",
@@ -898,7 +898,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.8"
+   "version": "3.9.2"
   },
   "toc": {
    "base_numbering": 1,

